lib/routes/signin-codes.js

```diff
@@ -56,7 +55,7 @@ module.exports = (log, db, customs) => {
                     return request.emitMetricsEvent(`flow.continued.${result.flowId}`)
                   }
                 })
-                .then(() => ({ email: result.email }))
+                .then(() => { return { email: result.email } })
```

Comment:
Why did this line need to change? Looks like they both do exactly the same thing?